src/components/ReportActionItem/MoneyReportView.js

```diff
@@ -35,11 +35,7 @@ function MoneyReportView(props) {
     return (
         <View>
             <View style={[StyleUtils.getReportWelcomeContainerStyle(props.isSmallScreenWidth), StyleUtils.getMinimumHeight(CONST.EMPTY_STATE_BACKGROUND.MONEY_REPORT.MIN_HEIGHT)]}>
-                <Image
-                    pointerEvents="none"
-                    source={EmptyStateBackgroundImage}
-                    style={[StyleUtils.getReportWelcomeBackgroundImageStyle(true)]}
-                />
+                <EmptyStateBackground />
```

Comment:
Can we also use this in the report action thread parent?

Reply:
@grgia Fixed!